mmv1/products/compute/OrganizationSecurityPolicy.yaml

```diff
@@ -55,10 +55,16 @@ parameters:
     required: true
     immutable: true
 properties:
-  - name: 'displayName'
+  - name: 'shortName'
     type: String
     description: |
-      User-provided name of the organization security policy. The name should be unique in the organization in which the security policy is created. This should only be used when SecurityPolicyType is FIREWALL.
+      User-provided name of the organization security policy. The name should be unique in the organization
```

Comment:
Looks like shortname was actually added in a separate PR in the meantime.

The plan is to just deprecate display_name here, right?